hotel-management/src/main.tsx

```diff
@@ -1,11 +1,14 @@
 import { StrictMode } from 'react';
 
```

Comment:
Why we have extra empty line here

Reply:
It's because of the rule 'import/order' from Eslint. It automatically formats for me.